docs/Quickstart.md

```diff
@@ -35,6 +35,15 @@ use_frameworks!
 target '<Your Target Name>' do
     pod 'SalemoveSDK', :git => 'https://github.com/salemove/ios-bundle'
 end
+
+post_install do |installer|
```

Comment:
Some more info on why this change is needed + ticket would be a good help for the reviewer.